src/Sparrow/Json/UnmanagedWriteBuffer.cs

```diff
@@ -15,6 +15,7 @@ public unsafe interface IUnmanagedWriteBuffer : IDisposable
         void WriteByte(byte data);
         void EnsureSingleChunk(JsonParserState state);
         void EnsureSingleChunk(out byte* ptr, out int size);
+        void Kill();
```

Comment:
I don't think this should be required

Reply:
We need it for the UnmanagedStreamBuffer.Kill, which should destroy it's ManagedPinnedBuffer

Reply:
That is NOT safe to do. The `ManagedPinnedBuffer` has its own finalizer

Reply:
ok